docs/example-narratives/temporary-ratings.md

```diff
@@ -0,0 +1,113 @@
+  * If for any reason it makes sense to temporarily update the recourse rating for a given resource.  
+
+## Creating Temporary Ratings
+Temporary ratings, whether they be Temporary AAR Exceptions or Seasonal Overrides, are created in 
+much the same manner.  The following `input.json` example could be used to create either a 
+Temporary AAR Exception or Seasonal Override either by invoking 
+[createTemporaryAARException](../spec#tag/Temporary%20AAR%20Exceptions/operation/createTemporaryAARException)
+or 
+[createSeasonalOverride](../spec#tag/Seasonal%20Overrides/operation/createSeasonalOverride)
+respectively:
+
+```json
+{% include_relative examples/temporary-aar-exception-post.json %}
+```
+
+Note that the structures for origin, the associated power system resources and its aliases, 
+as well as the rating values themselves all follow similar patterns and usages to AAR 
+proposal submissions.  
+
+### Creating a Temporary AAR Exception with `curl`
+
+Given the above `input.json`, run the following command to send it to the TROLIE server:
+
+```bash
+curl -d @input.json \
+-X POST \
+-H "Content-Type: application/vnd.trolie.temporary-aar-exception.v1+json" \
+-H "Accept: application/vnd.trolie.temporary-aar-exception.v1+json" \
+-o output.json \
+"https://trolie.example.com/temporary-aar-exceptions"
+```
+
+If this submission is successful, `output.json` will contain the contents of the
+response from TROLIE. This response is identical to the input, except that the TROLIE
+server will have assigned an ID to the exception.  An example of this response format 
+is given below:
+
+```json
+{% include_relative examples/temporary-aar-exception-get.json %}
+```
+
+## Updating and Deleting Temporary Ratings
+Once created, the temporary ratings may be updated.  This is often needed if the 
+anticipated end time changes, or if more detail needs to be added to the
+reason.  For the Temporary AAR Exception created above, this may be achieved by 
+invoking [updateTemporaryAARException](../spec#tag/Temporary%20AAR%20Exceptions/operation/updateTemporaryAARException).  
+
+Assuming the `output.json` file above was modified as desired, the Temporary
+AAR Exception may be updated with the following command.  Note that the ID
+returned from the create operation must be used in the URL:
+
+```bash
+curl -d @output.json \
+-X PUT \
+-H "Content-Type: application/vnd.trolie.temporary-aar-exception.v1+json" \
+-H "Accept: application/vnd.trolie.temporary-aar-exception.v1+json" \
+"https://trolie.example.com/temporary-aar-exceptions/46f7212b-1633-4c30-ba71-c6e987b2ded7"
+```
+
+Finally, the temporary rating may also have been created by mistake.  It may be 
+canceled by deleting it using 
+[deleteTemporaryAARException](../spec#tag/Temporary%20AAR%20Exceptions/operation/deleteTemporaryAARException)
+This can be done with the following command:
```

Comment:
One caveat here--it can't be canceled per se, if it is already underway. "Cancel" in that case would simply updated the end datetime to now.

Reply:
Added a little clarification to this point.